### ISSUE TITLE
Do you have example training curves?

### ISSUE THREAD

**Issue by alexander-soare** [hsinyilin19/ResNetVAE#2]
Thanks very much for sharing this great work. I think that if you add images of training curves to the README it could add a lot of value for people trying to replicate your results and verify that they are doing the right thing. It has been over a year since your last commit though, so totally understand if this is not top of mind!

**Comment by hsinyilin19** [comment]
Thanks for your suggestion. I've added the images of training curve to the README. Hope it helps!

**Comment by alexander-soare** [comment]
That's great! Thank you kindly

**Comment by hailuu684** [comment]
> Thanks for your suggestion. I've added the images of training curve to the README. Hope it helps!

Hi, I run the code, but the loss didn't start from 3 for cifar10, it starts from around this instead. Think you should use mean in loss function not sum
![image](https://github.com/hsinyilin19/ResNetVAE/assets/73364788/e39a5d44-eae1-4047-95ef-5cd7b9650b88)
